qiita_pet/templates/private_studies.html

```diff
@@ -134,6 +137,22 @@ <h4 class="modal-title" id="myModalLabel">Modify Sharing Settings</h4>
             {% end %}
         </tbody>
     </table>
+
+<!--User study Modals-->
+{% for s in user_studies %}
```

Comment:
Nope, it needs to be this way to allow dataframes to search over the abstracts. I can add the close button though.